alpenhorn/auto_import.py

```diff
@@ -239,6 +284,24 @@ def get_filecorrinfo_keywords_from_h5(path):
         "freq_number": freq_number,
     }
 
+    
+def get_filehfbinfo_keywords_from_h5(path):
```

Comment:
This function is never called.

You probably intended to call it before creating a new `HKPFileInfo` entry as part of the big if-elif clause down around here https://github.com/chime-experiment/alpenhorn/blob/e4685ef8dd42040f49960edc8f84ba1ee8cce7dd/alpenhorn/auto_import.py#L810